### ISSUE TITLE
colors.completion.odd.bg not being aplied on startup, but is on :config-source

### ISSUE THREAD

**Issue by lelgenio** [qutebrowser/qutebrowser#5783]
**Version info**:
qutebrowser v1.13.1
Git commit: f0f15b89b on master (2020-10-09 15:11:53 +0200)
Backend: QtWebEngine (Chromium 80.0.3987.163)
Qt: 5.15.1

**Does the bug happen if you start with `--temp-basedir`?**:
Impossible to tell, since the bug is on the reading of configuration files.
I deleted `.local/share/qutebrowser` and `.config/qutebrowser`, then created a config.py with only the affected line, the bug is still present.

**Description**
`colors.completion.odd.bg`  appears to be ignored when first opening qb, but works when sourcing the configuration.
The bug is also present for `autoconfig.yaml`.

**How to reproduce**
1. add `c.colors.completion.odd.bg = "#ff0000"` to `~/.config/qutebrowser/config.py`
2. start qutebrowser
3. observe odd completions **are not** red
4. run `:config-source`
5. odd completions are now red

**No idea why, but this "fixes" it**
1. add `c.colors.completion.odd.bg = "#ff0000"` to `~/.config/qutebrowser/config.py`
2. add a syntax error to `~/.config/qutebrowser/config.py`
3. start qutebrowser
4. ignore the error message
5. observe odd completions **are** red

**Comment by The-Compiler** [comment]
Works fine here.

Can you reproduce by doing:

`qutebrowser --temp-basedir -s colors.completion.odd.bg "#ff0000"`

If not, what about the following?

- Start `qutebrowser --basedir /tmp/completion`
- Quit it again
- `echo 'c.colors.completion.odd.bg = "#ff0000"' > /tmp/completion/config/config.py`
- Start `qutebrowser --basedir /tmp/completion`
- Press `:` to show the completion

**Comment by lelgenio** [comment]
Notice the color on startup is `#242424`, not the default value `#444444`, so running `:config-source` even with an empty `config.py` has this behaviour

**Comment by The-Compiler** [comment]
Do you use something like a custom Qt theme (what is `QT_QPA_PLATFORMTHEME` set to?) and/or qt5ct?

**Comment by lelgenio** [comment]
Yes `env QT_QPA_PLATFORMTHEME="" qutebrowser` does not have this issue

**Comment by The-Compiler** [comment]
So what theme are you using precisely, and what about the reproduction steps above?

**Comment by The-Compiler** [comment]
Can you please share your `~/.config/qt5ct/qt5ct.conf` then? I'd still like to reproduce this and either report it to the theme author or find some workaround.

**Comment by lelgenio** [comment]
`qutebrowser --temp-basedir -s colors.completion.odd.bg "#ff0000"`
![2020-10-10-160443_screenshot](https://user-images.githubusercontent.com/31388299/95663220-c5751180-0b13-11eb-830b-b49ecfa22561.png)

**Comment by lelgenio** [comment]
`.config/qt5ct/qt5ct.conf`
```
[Appearance]
color_scheme_path=/usr/share/qt5ct/colors/airy.conf
custom_palette=false
standard_dialogs=default
style=kvantum-dark

[Fonts]
fixed=@Variant(\0\0\0@\0\0\0\x16\0\x44\0\x65\0j\0\x61\0V\0u\0 \0S\0\x61\0n\0s@(\0\0\0\0\0\0\xff\xff\xff\xff\x5\x1\0\x32\x10)
general=@Variant(\0\0\0@\0\0\0\x16\0\x44\0\x65\0j\0\x61\0V\0u\0 \0S\0\x61\0n\0s@(\0\0\0\0\0\0\xff\xff\xff\xff\x5\x1\0\x32\x10)

[Interface]
activate_item_on_single_click=1
buttonbox_layout=0
cursor_flash_time=1000
dialog_buttons_have_icons=1
double_click_interval=400
gui_effects=@Invalid()
keyboard_scheme=2
menus_have_icons=true
show_shortcuts_in_context_menus=true
stylesheets=@Invalid()
toolbutton_style=4
underline_shortcut=1
wheel_scroll_lines=3

[SettingsWindow]
geometry=@ByteArray(\x1\xd9\xd0\xcb\0\x3\0\0\0\0\x5V\0\0\0\x14\0\0\t\t\0\0\x4\x1f\0\0\x5V\0\0\0\x14\0\0\b4\0\0\x3\x2\0\0\0\0\x2\0\0\0\a\x80\0\0\x5V\0\0\0\x14\0\0\t\t\0\0\x4\x1f)
```

**Comment by The-Compiler** [comment]
Still no luck at reproducing it I'm afraid, even with the exact same `qt5ct.conf`... What distribution is this on?

**Comment by lelgenio** [comment]
changing style from kvantum to Fusion also fixes the problem

**Comment by lelgenio** [comment]
I am on archlinux, i imagine this problem would happen is some distros wich ship kvantum by default, like manjaro

**Comment by The-Compiler** [comment]
If it'd show up on Manjaro by default I'd expect other people to have reported it as well. I'm on Archlinux as well and I can't reproduce it, no idea what could be different on your machine... What does `pacman -Q qt5ct qt5-base kvantum-qt5` say?

**Comment by lelgenio** [comment]
```
$ pacman -Q qt5ct qt5-base kvantum-qt5
qt5ct 1.1-1
qt5-base 5.15.1-3
kvantum-qt5 0.16.1-1
```

**Comment by The-Compiler** [comment]
Same here... I'm out of ideas - if someone else who uses qt5ct wants to install kvantum and see what happens, I'd be happy about some help with trying to reproduce this!

**Comment by lelgenio** [comment]
Seems to be caused by `KvantumManager > Configure Active Theme > Hacks > Respect Dark Themes`
Now i am conflicted, this is a **hack**, yet it comes enabled by default on some themes shiped with Kvantum, like KvArcDark
Still now idea why it only shows up on `colors.completion.odd`

fixing this on qb's side may be unecessary

**Comment by The-Compiler** [comment]
I still can't reproduce with that enabled and the `KvArcDark` theme. Can you please share `~/.config/Kvantum/kvantum.kvconfig` as well as `~/.config/Kvantum/<Themename>/<Themename>.kvconfig`?

**Comment by lelgenio** [comment]
```
$ cat .config/Kvantum/kvantum.kvconfig
[General]
theme=KvArcDark

$ tree .config/Kvantum/
.config/Kvantum/
└── kvantum.kvconfig

```
There is nothing else on `.config/Kvantum`, since i deleted everything under it, changed the theme to KvArcDark and verified the issue is still present.

**Comment by bitraid** [comment]
I don't know if this is somehow related, but i've also noticed some issues with completion background colors, that are reproducible across different OS/versions:

Starting qutebrowser with `--temp-basedir`:
- `:set colors.completion.odd.bg 'rgba(0, 43, 54, 0.8)'`
changes the color but doesn't set alpha. same goes for `colors.completion.category.bg`.
- `:set colors.completion.even.bg 'rgba(0, 43, 54, 0.8)'`
doesn't get the correct color but the alpha is set and now is also set for `odd.bg` (and `category.bg` if was previously set).

There are also issues when gradient colors are used for `odd.bg`, `even.bg`.
